contents/data_vis_makie_glmakie.md

```diff
@@ -1,15 +1,15 @@
 ## GLMakie.jl {#sec:glmakie}
 
-`CairoMakie.jl` supplies all our needs for static 2D images.
+`CairoMakie.jl` fulfills all our needs for static 2D images.
 But sometimes we want interactivity, especially when we are dealing with 3D images.
 Visualizing data in 3D is also a common practice to gain insight from your data.
-This is where `GLMakie.jl` might be helpful, since it uses [OpenGL](http://www.opengl.org/) as a backend that adds interactivity and responsiveness to plots.
+This is where `GLMakie.jl` comes into play, since it uses [OpenGL](http://www.opengl.org/) as a backend that adds interactivity and responsiveness to plots.
 Like before, a simple plot includes, of course, lines and points. So, we will start with those and since we already know how layouts work, we will put that into practice.
 
 ### Scatters and Lines
 
 For scatter plots we have two options, the first one is `scatter(x, y, z)` and the second one is `meshscatter(x, y, z)`.
-In the first one markers don't scale in the axis directions, but in the latter they do because they are actual geometries in 3D space.
+In the former markers don't scale in the axis directions and in the latter they do because they are actual geometries in 3D space.
```

Comment:
```suggestion
In the former, markers don't scale in the axis directions and in the latter they do because they are actual geometries in 3D space.
```
I first read it as, "In the former markers, don't scale ..." and got confused.